python/lib/api.py

```diff
@@ -0,0 +1,79 @@
+"""Allows LORIS API connectivity for LORIS-MRI python code base"""
```

Comment:
This is fine for now but I am guessing that as we use more API functions for querying, this file is going to become gigantic. No big deal for now but we might want to think about a proper structure for API querying (a bit like now there is a database folder handling database queries with several files depending on what is being queried). Just a thought.